src/components/infopopups/common/infopopup.css

```diff
@@ -151,7 +151,7 @@
     border-left: solid 7px #f2f2f2;
     padding-left: 30px;
     margin-left: 42px;
-    margin-top: 7px;
+    margin-top: 0px;
```

Comment:
this surprises me... I cannot see whether this is vertical-line class, because GitHub code review views suck... but I thought we wanted the space here so the ... oh wait, padding on the div I'm betting. nm